src/plat/bcm2711/overlay-rpi4.dts

```diff
@@ -17,8 +18,83 @@
 		    &{/timer};
 	};
 
+	/*
+	 * See BCM2711 TRM section 1.2 and
+	 * https://github.com/raspberrypi/firmware/issues/1374 for
+	 * reference.
+	 *
+	 * RPi4B seems to be in "low peripheral" mode by default.
```

Comment:
Good to have this written down finally, just remove the "seems" and say "runs" because it looks like this is a fact.

Reply:
Comment fixed to state this as a fact.